client/jetpack-connect/signup.js

```diff
@@ -151,7 +156,14 @@ export class JetpackSignup extends Component {
 		debug( 'submitting new account', userData );
 		this.setState( { isCreatingAccount: true }, () =>
 			this.props
-				.createAccount( { ...userData, extra: { ...userData.extra, jpc: true } } )
+				.createAccount( {
+					...userData,
+					signup_flow_name: this.getFlowName(),
```

Comment:
Now we re setting this prop explicitly but I can't see it was set to `jetpack-connect` as a flow name here before. Is there a  risk breaking something here?

Reply:
`getFlowName()` returns either `woodna:whatever` *or* `jetpack-connect`, so for non-WooDNA the flow will still be `jetpack-connect`. In the server, I searched for all the usages of `"jetpack-connect"` and made sure that if the flow is `woodna:*` the code follows the same path (you'll see when you get to the server diff :) ).